chainerx_cc/chainerx/cuda/cuda_device/reduction.cu

```diff
@@ -131,6 +131,15 @@ public:
 
 CHAINERX_CUDA_REGISTER_KERNEL(SumKernel, CudaSumKernel);
 
+class CudaCumsumKernel : public CumsumKernel {
+public:
+    void Call(const Array& a, int8_t axis, const Array& out) override {
```

Comment:
Could you comment out these arguments to avoid unused-parameter error?
```suggestion
    void Call(const Array& /* a */, int8_t /* axis */, const Array& /* out */) override {
```

Reply:
Done.